src/App.tsx

```diff
@@ -3,24 +3,24 @@ import logo from './logo.svg';
 import './App.css';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	return (
```

Comment:
I've always been a 4 space/tab kinda guy like what is done here, I am just wondering, does this seem excessively spaced? Have I been wrong all along? It always looks different in an editor than GitHub though so, I am not sure